Code/GraphMol/ForceFieldHelpers/UFF/Builder.h

```diff
@@ -89,7 +91,7 @@ void addNonbonded(const ROMol &mol, int confId, const AtomicParamVect &params,
                   bool ignoreInterfragInteractions = true);
 void addTorsions(const ROMol &mol, const AtomicParamVect &params,
                  ForceFields::ForceField *field,
-                 std::string torsionBondSmarts = "[!$(*#*)&!D1]~[!$(*#*)&!D1]");
+                 std::string torsionBondSmarts = defaultTorsionBondSmarts);
```

Comment:
It's probably still worth doing since it won't break any calling code and it's definitely the right thing to do.